packages/ui-pagination/src/Pagination/PaginationButton/index.tsx

```diff
@@ -56,12 +56,14 @@ class PaginationButton extends Component<PaginationPageProps> {
 
   render() {
     const exclude = this.props.current ? ['onClick', 'href'] : []
+
     const props = omitProps(this.props, PaginationButton.allowedProps, exclude)
+
     return (
       <BaseButton
         color="primary"
-        withBackground={this.props.current ? true : false}
-        withBorder={this.props.current ? true : false}
+        withBackground={!!this.props.current}
+        withBorder={!!this.props.current}
```

Comment:
I don't really understand the need for the previous version of this code, but why is this `!!` needed?
Since `props.current` is a boolean why are we double negating it?

Reply:
Well there is no "need", but since TS cannot really handle default prop values, it was `boolean | undefined` type... Which is still a totally valid value for the withBackground/withBorder prop, but wanted to indicate that it has a default value, and I didn't like `withBackground={this.props.current!}` :D I guess I'll just remove it, since it is really not needed...

Reply:
removed it :)